### DISCUSSION TITLE
Remirror & Yjs (via WebRTC) in a serverless architecture PoC

### DISCUSSION THREAD

**Discussion by whawker** [remirror/remirror#958] [Show and tell]
We want to run Remirror with Yjs in a serverless architecture. 

Sharing this here in the hope that others can benefit from this as well: https://github.com/Collaborne/remirror-yjs-webrtc-demo 😄 

The README describes the questions we aimed to answer with the PoC and the approach that we took.

I'd **highly** appreciate any feedback on the chosen approach! Does this reflect best practice?

**Comment by ifiokjr** [comment]
Wow! I really like this. I'm not an expert on collab editing, but I'll take a look soon and get back to you.
